python/tank/context.py

```diff
@@ -737,10 +737,133 @@ def from_entity(tk, entity_type, entity_id):
         
         context.update(entity_context)
 
+        if entity_type == "Project":
```

Comment:
The diff is a bit weird here but it was just indenting the test as it was previously outside the if-else even though it was only valid inside the else!